programs/zk-token-proof/src/lib.rs

```diff
@@ -134,6 +134,11 @@ declare_process_instruction!(process_instruction, 0, |invoke_context| {
 
     match instruction {
         ProofInstruction::CloseContextState => {
+            if native_programs_consume_cu {
+                invoke_context
+                    .consume_checked(3_300)
```

Comment:
nit: perhaps can define those hardcoded number as const, like [this](https://github.com/solana-labs/solana/blob/master/programs/bpf_loader/src/lib.rs#L65-L67), so the default cost can be referenced elsewhere like [here](https://github.com/solana-labs/solana/blob/master/cost-model/src/block_cost_limits.rs#L41-L43).